packages/rainbowkit/src/components/RainbowKitProvider/AvatarContext.ts

```diff
@@ -0,0 +1,18 @@
+import React, { createContext } from 'react';
+
+export type AvatarComponentProps = {
+  address: string;
+  ensImage?: string | null;
+  size: number;
+};
+export type AvatarComponent = React.FunctionComponent<AvatarComponentProps>;
+
+export const defaultAvatar = {
+  component: undefined,
```

Comment:
To reduce code paths, could we make an EmojiAvatar component and use it as the default value? This way we can always just render the AvatarComponent from context, even when it's our own.